comparison.js

```diff
@@ -0,0 +1,98 @@
+const path = require('path');
+var fs = require('fs');
+var browserify = require('browserify-middleware');
+var express = require('express');
+var minimist = require('minimist');
+var _ = require('lodash');
+
+
+var argv = minimist(process.argv.slice(2));
+var serverPort = argv['port'] || 8080;
+var annotationFolders = argv['_'];
+if (annotationFolders.length == 0) {
+    annotationFolders = [__dirname + '/public/comparison/prediction', __dirname + '/public/comparison/gold'];
+}
+
+
+// Retrieve file list for every request might be slow, while allowing adding new files without restarting the server
```

Comment:
As written, this comment is ambiguous: what decision was made here OR how does this method support a choice by the user/client?